src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -365,15 +365,16 @@ LedgerMaster::getFullValidatedRange (std::uint32_t& minVal, std::uint32_t& maxVa
     if (!maxVal)
         return false;
 
+    boost::optional<std::uint32_t> maybeMin;
```

Comment:
If anything, I think changing this to _return_ `boost::optional<std::pair<std::uint32_t,std::uint32_t>>` makes better sense.